test/json_logic_test.exs

```diff
@@ -1,10 +1,43 @@
 defmodule JsonLogicTest do
-  use ExUnit.Case
+  use ExUnit.Case, async: true
   doctest JsonLogic
 
-  test "apply" do
-    assert JsonLogic.apply(nil) == nil
-    assert JsonLogic.apply(%{}) == %{}
+  describe "non rules" do
+    test "nil" do
+      assert JsonLogic.apply(nil) == nil
```

Comment:
These `apply` calls are incompatible with the change I made earlier to swap to the name `resolve` so it doesn't conflict with `Kernel.apply`